lib/ansiblelint/cli.py

```diff
@@ -46,17 +46,33 @@ def expand_to_normalized_paths(config, base_dir=None):
 
 
 def load_config(config_file):
-    config_path = config_file if config_file else ".ansible-lint"
-    config_path = os.path.abspath(config_path)
-
-    if not os.path.exists(config_path):
+    config_path = os.path.abspath(config_file or '.ansible-lint')
+
+    if config_file:
+        if not os.path.exists(config_path):
+            print(
+                "Config file not found '{cfg!s}'.".format(cfg=config_path),
+                file=sys.stderr,
+            )
+            sys.exit(2)
```

Comment:
Could you pls move `2` to a global constant like `INVALID_CONFIG_RC = 2`?
```suggestion
            sys.exit(INVALID_CONFIG_RC)
```

I think it'd communicate the intent of the magic literal better. And would improve the consistency too.